code/game/gamemodes/dynamic/dynamic_storytellers.dm

```diff
@@ -39,6 +39,23 @@ Property weights are:
 	var/event_injection_cooldown_middle = 0.5*(GLOB.dynamic_event_delay_max + GLOB.dynamic_event_delay_min)
 	mode.event_injection_cooldown = (round(CLAMP(EXP_DISTRIBUTION(event_injection_cooldown_middle), GLOB.dynamic_event_delay_min, GLOB.dynamic_event_delay_max)) + world.time)
 
+/datum/dynamic_storyteller/proc/calculate_threat()
+	var/threat = 0
+	for(var/datum/antagonist/A in GLOB.antagonists)
```

Comment:
(it's like this everywhere else in the code and i have no idea if it's absolutely necessary but didn't want to risk it--this vs var/a in GLOB.antagonists and then typecasting the next line)